packages/components/src/internal/utils.ts

```diff
@@ -1,3 +1,5 @@
+// TODO: All of these methods should be moved to the editable folder.
```

Comment:
nit: Seems like as good a time as any to do this.

Reply:
Oh for sure, but I think it's going to be the last thing I do in this PR, that way the diffs are easier to understand during review, and so I can isolate any issues the move caused.